packages/shared/src/broker.ts

```diff
@@ -82,12 +89,6 @@ const validateRequest = (network: ChainflipNetwork, params: unknown) => {
     .superRefine(ensureDcaWithFok)
     .parse(params);
 
-  if (parsed.destAsset.chain === 'Polkadot') {
```

Comment:
we sure that we can remove this? 😬

Reply:
yeas, a new case has been added to the union where we transform the ss58 address into hex